lib/cc/cli/command.rb

```diff
@@ -43,6 +43,10 @@ def require_codeclimate_yml
         end
       end
 
+      def set_local_owner
```

Comment:
Is there a better name here? The purpose of the method isn't really to "set local owner" (it sets owner and group, and there's only a loose concept of "local"). What about `#correct_permissions`?